giskard_vision/core/detectors/base.py

```diff
@@ -169,3 +169,13 @@ def get_results(self, model: Any, dataset: Any) -> List[ScanResult]:
             List of ScanResult objects containing evaluation results
         """
         ...
+
+
+EthicalIssueMeta = IssueGroup(
+    "Ethical",
+    description="The data are filtered by metadata like age, facial hair, or gender to detect ethical biases.",
+)
+PerformanceIssueMeta = IssueGroup(
+    "Performance",
+    description="The data are filtered by metadata like emotion, head pose, or exposure value to detect performance issues.",
+)
```

Comment:
these are now in `dataloaders/base.py`, they also belong nicely here, but as the issue groups are relevant for dataloaders when setting meta, i would leave it the dataloaders base, could you refactor this please?

Reply:
ofc

Reply:
The user can call the method in the super class to get the basic metadata, and then filter them or change the issue group during the merge of the dataset-specific ones.